src/test/java/org/folio/rest/impl/OrdersResourceImplTest.java

```diff
@@ -125,23 +126,35 @@ public void testPoCreationFailure(TestContext ctx) throws Exception {
 
     String body = getMockData(poCreationFailurePath);
 
-    final Response resp = RestAssured
-      .with()
-      .header(X_OKAPI_URL)
-      .header(X_OKAPI_TENANT)
-      .contentType(APPLICATION_JSON)
-      .body(body)
-      .post(rootPath)
-      .then()
-      .contentType(TEXT_PLAIN)
-      .statusCode(400)
-      .extract()
-      .response();
+    final Async asyncLocal = ctx.async();
```

Comment:
Is this needed?  If the rest assured call is synchronous (I think they are), then which part of this is asynchronous?

Reply:
Sorry, habit when dealing with vertx tests. :) Agreed, while it doesn't hurt, in this case it is not needed. I'll remove it.